infrastructure/src/main/java/org/corfudb/infrastructure/management/ReconfigurationEventHandler.java

```diff
@@ -69,12 +69,9 @@ public static boolean handleFailure(@Nonnull IReconfigurationHandlerPolicy failu
      */
     private static Duration getStateTransferTimeoutEstimate(CorfuRuntime runtime) {
 
-        // Try to estimate a reasonable timeout to rebuild the logging unit
-        Token trimMark = runtime.getAddressSpaceView().getTrimMark();
-
         long tail = runtime.getAddressSpaceView().getLogTail();
 
-        long rangeToReplicate = tail - trimMark.getSequence();
+        long rangeToReplicate = tail - 0L;
```

Comment:
-0L? we can remove this.

Reply:
I don't know why Wenbin's change is in this PR. @WenbinZhu Could you help answer this question?

Reply:
This is not my change.

Reply:
I have deleted 0L for conciseness.